src/akushon/action/model/action.cpp

```diff
@@ -110,4 +110,19 @@ void Action::reset()
   poses.clear();
 }
 
+void Action::map_action(const Action & source_action, const Action & target_action, int target_pose_index, float source_val, float source_min, float source_max)
+{
+  std::vector<tachimawari::joint::Joint> src_joints = source_action.get_pose(target_pose_index).get_joints();
+  std::vector<tachimawari::joint::Joint> target_joints = target_action.get_pose(target_pose_index).get_joints();
+  for (int joint = 0; joint < 20; joint++)
```

Comment:
Prefer prefix increment over postfix increment:
```suggestion
  for (int joint = 0; joint < 20; ++joint)
```

If the iteration order doesn't matter, it may be better to use decrement over increment (see [this](https://stackoverflow.com/questions/8968785/which-loop-has-better-performance-increment-or-decrement)):
```suggestion
  for (int joint = 19; joint >= 0; --joint)
```